tile.cpp

```diff
@@ -1358,9 +1358,15 @@ unsigned long long choose_mingap(std::vector<unsigned long long> const &indices,
 	return top;
 }
 
-long long choose_minextent(std::vector<long long> &extents, double f) {
```

Comment:
A comment explaining the logic here would be nice -- specifically what are the "extents" and what's the "fraction" represent conceptually.
Is `off` just short for `offset`? If so spelling it out seems a bit more legible

Reply:
Thanks, I'll add a comment.

The "extents" are the areas of polygons or the pseudo-areas associated with linestrings and points. The "fraction" is the proportion of the features that tippecanoe is trying to preserve in the hope of making the tile small enough, so when it says

```
Going to try keeping the biggest 20.19% of the features to make it fit
```

the fraction is 0.2019. And yes, "off" is the array index of the "extent" that it hopes will be good enough to make the tile the right size.

Reply:
Comment added